Validation/Configuration/python/hgcalSimValid_cff.py

```diff
@@ -10,6 +10,7 @@
 from Validation.HGCalValidation.digiValidation_cff      import *
 from Validation.HGCalValidation.rechitValidation_cff    import *
 from Validation.HGCalValidation.hgcalHitValidation_cfi  import *
+from RecoHGCal.TICL.SimTracksters_cff import *
```

Comment:
Should this `import` rather go in `Validation/Configuration/python/globalValidation_cff.py`, where `ticlSimTrackstersTask` is used?

Reply:
I would like to keep the number HGCAL-related includes in `globalValidation_cff.py` limited, so that one knows that everything related to HGCAL comes from `hgcalSimValid_cff.py`